materials/gganimate.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: page
+element: notes
+title: gganimate
+language: R
+---
+
+### Installation
+
+* Some R packages are not published to CRAN
+* Can install from other places using `devtools`
+
+```
+install.packages("devtools")
+devtools::install_github("dgrtwo/gganimate")
+```
+
+### Example
+
+* Example using weather data
+* Get this data using `rnoaa`
+
+```
+install.packages('rnoaa')
+library('rnoaa')
+gnv_weather <- meteo_pull_monitors("USW00012816", date_min = "1984-01-01", date_max = "2016-12-31")
+```
+* Plot annual temperature pattern through time
+* First make a plot of the annual temperature trend using ggplot
+
+```
+library(ggplot2)
+library(gganimate)
+
+gnv_weather$year = format(gnv_weather$date, "%Y")
+gnv_weather$jday = format(gnv_weather$date, "%j")
```

Comment:
If the jday column is changed to numeric (e.g., `gnv_weather$jday = as.numeric(format(gnv_weather$date, "%j"))`), the x-axis on the resulting plot has reasonably spaced labels.